app/src/main/java/com/teamagam/gimelgimel/app/common/logging/LoggerFactory.java

```diff
@@ -16,15 +22,15 @@
     private static final NativeLogger sInnerLogger =
             new NativeLogger(LoggerFactory.class.getSimpleName());
 
-    private static FileWriter sLogWriter;
+    private static String sExternalStorageDirectoryPath;
 
     public static void init(Context context) {
-        try {
-            sLogWriter = DiskLogger.createLogfileWriter(context);
-        } catch (DiskLogger.LogfileCreationException ex) {
-            sInnerLogger.e("Initializing disk-logger failed", ex);
-            sLogWriter = null;
-        }
+        sExternalStorageDirectoryPath = getExternalStorageDirectoryPath(context);
```

Comment:
I'd put that initializing code inside a try-catch
Its creating directory, right ? !fix